spring-cloud-aws-samples/spring-cloud-aws-sns-sample/infrastracture/README.md

```diff
@@ -0,0 +1,52 @@
+# Spring Cloud AWS SNS Sample App Infrastructure
+
+Infrastructure code to run **Spring Cloud AWS SNS Sample App** based on [AWS CDK](https://aws.amazon.com/cdk/)
+
+## How to install
+
+* `npm install`
+
+## How to run
+
+* `cdk synth`
+* `cdk deploy`
+
+Once you don't need the infrastructure anymore, it can be destroyed by calling:
+
+* `cdk destroy`.
+
+You will need following policies to run cdk and Sample app:
+(Ideally this should be two different policies. One just for SNS and another for cloudFormations. For sake of sample and simplicity we will use one.)
+
```

Comment:
Since it's a markdown the IAM policy should be formatted with backticks https://www.markdownguide.org/basic-syntax/#code-blocks